docs/Superoperator representations.md

```diff
@@ -12,49 +16,565 @@ where $i$ is a row and $j$ is a column index.
+
+
+## The $n$-qubit Pauli basis 
+ The $n$-qubit Pauli basis is denoted $\mathcal P^{\otimes n} $ where $\mathcal  P = \{ I, X, Y, Z \}$ are the usual Pauli matricies. It is an operator baisis for the $d = 2^n$ dimensional Hilbert space and there are $d^2 = 4^n$ operators in $\mathcal P^{\otimes n} $. If one divides all the operators by $\sqrt{d}$ the basis is orthonormal with respect to the Hilbert-Schmidt inner product.
+ 
+It is often convenient to index the $d^2$ operators with a single label, e.g. $P_1=I^{\otimes n},\, \ldots,\, P_{d^2}= Z^{\otimes n}$  (or $P_0=I^{\otimes n}$ if you like zero indexing). In anycase, as these operators are Hermitian and unitary they obey $P_i^2=I^{\otimes n}$.
+ 
+To be explicit, for two qubits $d=4$ and we have 16 operators e.g. $\{II, IX, IY, IZ, XI, XX, XY, ..., ZZ\}$ were $II$ should be intepreted as $I\otimes I$ etc. The single index would be $\{P_1, P_2, P_3, P_4, P_5, P_6, P_7, ..., P_{16}\}$.
+
+## Quantum channels in the Kraus decomposition (or operator-sum representation)
+A completely positive map on the state $\rho$ can be written using a set of Kraus operators $\{ M_k \}$ as
+
+
+$$\rho' =\mathcal E (\rho) = \sum_{k=1}^N M_k \rho M_k^\dagger, $$
+where $\rho'$ is the state at the output of the channel.
+
+If $\sum_k M_k^\dagger M_k= I $ the map is trace preserving. It turns out that $N\le d^2$ where $d$ is the Hilbert space dimension e.g. $d=2^n$ for $n$ qubits.
+
+
+## Kraus to process matrix (aka $\chi$ or chi matrix)
+We choose to represent the process matrix in the Pauli basis. So we expand each of the Kraus operators in the $n$ qubit Pauli basis 
+
+$M_k = \sum^{d^2}_{j=1}c_{kj}\,P_j$ 
+
+where $\mathcal P_j \in \mathcal P ^{\otimes n}$.
+
+Now the channel $\mathcal E$ can be written as
+
+$\mathcal E (\rho) = \sum_{i,j=1}^{d^2} \chi_{i,j} P_i\rho P_j ,$
+
+where 
+
+$$\chi_{i,j} = \sum_k c_{k,i} c_{k,j}^*$$ 
+
+is an element of the process matrix $\chi$  of size $d^2 \times d^2$. The if the channel is CP the process matrix is a Hermitian and positive semidefinite. 
+
+
+## Kraus to Pauli Transfer matrix
+We begin by defining the Pauli vector representation of the state $\rho$ 
+
+$$ |\rho \rangle \rangle = \sum_j c_j |P_j\rangle \rangle$$
+
+where $P_j \in \mathcal P^{\otimes n}$ and $c_j = (1/d) \langle\langle P_j|\rho \rangle\rangle$.
+
+The Pauli-Liouville or Pauli transfer matrix representation of the channel $\mathcal E$ is denoted by $R_{\mathcal E}$. The matrix elements are
+
+$$(R_{\mathcal E})_{i,j} = \frac 1 d {\rm Tr}[P_i \mathcal E(P_j)].$$
+
+Trace preservation implies $(R_{\mathcal E})_{0,j} = \delta_{0,j}$, i.e. the first row is one and all zeros. Unitality implies $(R_{\mathcal E})_{i,0} = \delta_{i,0}$, the first column is one and all zeros.
+
+In this representation the channel is applied to the state by multiplication
+
+$$|\rho' \rangle \rangle = R_{\mathcal E} |\rho \rangle \rangle$$
+
 
 ## Kraus to Superoperator
```

Comment:
I don't like calling these things superoperator. For me, a superoperator is an abstract concept, while these representations in computers are typically concrete representations in a particular basis.  Given the usage in the NMR community (I can give references), calling it the Liouville representation seems reasonable to me.

Reply:
I get it. But I think this is a battle you wont win (against the community). I'll mention the tension, see if you are happy with the discussion.